src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -193,6 +193,10 @@ protected function registerMappingDrivers($objectManager, ContainerBuilder $cont
                     $args[0] = array_merge(array_values($driverPaths), $args[0]);
                 }
                 $mappingDriverDef->setArguments($args);
+            } elseif ('attribute' === $driverType) {
+                $mappingDriverDef = new Definition('%'.$this->getObjectManagerElementName('metadata.'.$driverType.'.class%'), [
```

Comment:
Are you sure this line is correct?

I suggest to not concat `$driverType`. 

So, should it really be:

```php
new Definition('%'.$this->getObjectManagerElementName('metadata.attribute.class%')
```

Or did you mean to write: 


```php
new Definition('%'.$this->getObjectManagerElementName('metadata.attribute.class').'%'
```

Reply:
I copied that from the annotation code. Yours makes more sense, but the method returns the string , so the result is the same either way. I will clean this up.